.github/workflows/test.yml

```diff
@@ -215,6 +215,135 @@ jobs:
           min_coverage: 20
           token: ${{ github.token }}
   
+  ##############################################################################
+  # JOB: UNIT TEST LOGIN-SERVER  ###############################################
+  ##############################################################################
+  unit_test_login_server:
+    name: Unit tests - Login-Server
+    runs-on: ubuntu-latest
+    needs: []
+    services:
+      mariadb:
+        image: gradido/mariadb:test
+        env:
+           MARIADB_ALLOW_EMPTY_PASSWORD: 1
+           MARIADB_USER: root
+        ports: 
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping"
+                 --health-interval=6s
+                 --health-timeout=3s
+                 --health-retries=4
+    steps: 
+      - name: Debug service
+        run: echo "$(docker ps)"
+      - name: Debug container choosing script
+        run: echo "$(docker container ls | grep mariadb | awk '{ print $1 }')"
+      ##########################################################################
+      # CHECKOUT CODE ##########################################################
+      ##########################################################################
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      ##########################################################################
+      # Build Login-Server Test Docker image ###################################
+      ##########################################################################
+      - name: login server | Build `test` image
+        run: |
+          docker build --target test -t "gradido/login_server:test" -f ./login_server/Dockerfile login_server/
```

Comment:
I assume that you like to test if Gradido_LoginServer is building.
For testing I need to build Gradido_LoginServer_Test which use the same source + additonal test src but don't need Gradido_LoginServer. So to speed up things I have make it separate so that building Gradido_LoginServer and building Gradido_LoginServer_Test can run in parallel. So instead 16 Minutes it needs only 8 Minutes to run everything.

Reply:
But you dont reuse the image? I dont understand?

Reply:
No I don't reuse the image. 
The second reason is because it is nearly 1 GB in size and sometimes uploading that take ages.